pydicom/pixel_data_handlers/rle_handler.py

```diff
@@ -0,0 +1,219 @@
+import sys
+import pydicom.uid
+import pydicom.encaps
+from struct import unpack
+have_numpy = True
+try:
+    import numpy
+except ImportError:
+    have_numpy = False
+    raise
+
+sys_is_little_endian = (sys.byteorder == 'little')
+
+RLESupportedTransferSyntaxes = [
+    pydicom.uid.RLELossless,
+]
+
```

Comment:
I think they had two different meanings - 1st are the Syntaxes grouped by type. 2nd are the syntaxes that are supported by the pixel_handler (e.g. your JPEG handler may not handle all JPEGs). A bit pointless for a division in this case - bit I did it to mirror the other handlers.